how-to/containerd-kata.md

```diff
@@ -0,0 +1,357 @@
+  ```yaml
+  annotations:
+     io.kubernetes.cri.untrusted-workload: "true"
+  ```
+- Similarly, CRI-O introduced the annotation `io.kubernetes.cri-o.TrustedSandbox` for the same case.
+
+To eliminate the complexity of user configuration introduced by the non-standardized annotations and provide 
+extensibility, `RuntimeClass` was introduced and users could affect the runtime behavior through it without 
+the knowledge of the CRI daemons. We suggest the users have multiple runtimes use `RuntimeClass` instead of 
+the deprecated annotations.
+
+### Containerd Runtime V2 API (Shim V2 API)
+
+The [`containerd-shim-kata-v2` (short as `shimv2` in this documentation)](https://github.com/kata-containers/runtime/tree/master/containerd-shim-v2) 
+implements the [Containerd Runtime V2 (Shim API)](https://github.com/containerd/containerd/tree/master/runtime/v2) for Kata.
+With `shimv2`, kubernetes could launch Pod and OCI compatible containers with one shim (the `shimv2`) per Pod instead
+of `2N+1` shims (a `containerd-shim` and a `kata-shim` for each container and the Pod sandbox itself), and no standalone
+`kata-proxy` process even if no vsock available.
+
+![kubernetes integration with shimv2](../arch-images/shimv2.svg)
+
+The shim v2 is introduced in containerd [v1.2.0](https://github.com/containerd/containerd/releases/tag/v1.2.0) and kata shimv2
+is implemented in Kata Containers v1.5.0.
+
+## Install
+
+### Install Kata Containers
+
+Follow the instructions to [install Kata Containers](https://github.com/kata-containers/documentation/blob/master/install/README.md).
+
+### Install containerd with cri plugin
+
+> **Note:** `cri` is a native plugin of containerd 1.1 and above. It is built into containerd and enabled by default. 
+> You don't need to install `cri` if you have containerd 1.1 or above. Just check if the `cri` plugin has been disabled 
+> in the containerd configuration file.
+
+Follow the instructions from the [CRI installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md).
+
+And check if `containerd` is now available:
+  
+```bash
+$ command -v containerd
+```
+
+### Install CNI plugins
+
+> **Note:** You don't need install CNI plugins if you don't want to use containerd with Kubernetes.
+> And if you have installed Kubernetes with `kubeadm`, you may have already had CNI plugins installed.
+
+You may manually install CNI plugins as follows:
+
+```bash
+$ go get github.com/containernetworking/plugins
+$ pushd $GOPATH/src/github.com/containernetworking/plugins
+$ ./build_linux.sh
+$ sudo mkdir /opt/cni
+$ sudo cp -r bin /opt/cni/
+$ popd
+```
+
+### Install cri-tools
+
+> **Note:** `cri-tools` is a set of tools for CRI for the development and testing. For users only want to use containerd
+> with kubernetes, you may skip the `cri-tools`.
+
+You may install the `cri-tools` from source code:
```

Comment:
Suggested rewrite: 
You can install the `cri-tools` from source code as follows: